docs/_common/cluster-setup-for-visual-tutorials.md

```diff
@@ -0,0 +1,95 @@
+Below are instructions for setting up a Kubernetes cluster with network policies.
+If you don't have a cluster already, we recommend starting out with a Minikube cluster.
+
+<Tabs groupId="cni">
+<TabItem value="minikube" label="Minikube">
+
+If you don't have the Minikube CLI, first [install it](https://minikube.sigs.k8s.io/docs/start/). 
+
+Then start your Minikube cluster with Calico, in order to enforce network policies.
+
+```bash
+minikube start --cpus=4 --memory 8192 --disk-size 32g --cni=calico
```

Comment:
I don't think we should have two sets of instructions, one of which doesn't work for some of the tutorials -- let's add the relevant flags to the base set of instructions.